manifest.json

```diff
@@ -205,6 +212,13 @@
           "description": "logo_description",
           "label": "logo_label"
         },
+        {
+          "identifier": "show_brand_name",
+          "type": "checkbox",
+          "description": "show_brand_name_description",
+          "label": "show_brand_name_label",
+          "value": true
```

Comment:
for new settings the recommendation is to first add it with a default value of what exists in current themes (in this case false) and then swap the value after an update has been delivered to all accounts. Otherwise existing themes will start showing unexpected new content 😬

Reply:
OK I will change this to false as a temporary value for the rollout.